crawl/workspace.py

```diff
@@ -31,20 +31,10 @@ def __init__(self, repo_root_path, config, maven_install_info,
         self.dependency_metadata = dependency_metadata
         self.change_detection_enabled = config.change_detection_enabled
         self.override_file_info = override_file_info
-        self._name_to_ext_deps = self._parse_maven_install(maven_install_info, repo_root_path)
+        self.label_to_ext_deps = {}
```

Comment:
Minor: let's call this label_to_ext_dep, not plural (since we are not mapping to a container of deps but a single dep)